core/src/main/java/org/everit/json/schema/internal/TemporalFormatValidator.java

```diff
@@ -26,7 +26,7 @@
 
     TemporalFormatValidator(DateTimeFormatter formatter, List<String> formatsAccepted) {
         this.formatter = requireNonNull(formatter, "formatter cannot be null");
-        this.formatsAccepted = new ArrayList<>(formatsAccepted);
+        this.formatsAccepted = Collections.unmodifiableList(formatsAccepted);
```

Comment:
`Collections.unmodifiableList()` just returns an immutable view of the original array, but doesn't copy it, therefore here if the list gets mutated on the caller side, then the change will be reflected in `this.formatsAccepted` . The proper defensive way of this copy without guava is `this.formatsAccepted = unmodifiableList(new ArrayList<>(formatsAccepted))`.